idea/src/org/jetbrains/kotlin/idea/inspections/NullChecksToSafeCallInspection.kt

```diff
@@ -78,7 +81,7 @@ private fun collectNullCheckExpressions(expression: KtBinaryExpression): Triple<
 
 private fun isNullChecksToSafeCallFixAvailable(expression: KtBinaryExpression): Boolean {
     val (lte, rte) = collectNullCheckExpressions(expression) ?: return false
-    return getDuplicates(lte, rte)?.parent as? KtDotQualifiedExpression != null
+    return lte.isChainStable() && getDuplicates(lte, rte)?.parent as? KtDotQualifiedExpression != null
```

Comment:
You can check the case when `a` is nullable extension property. Also, you can modify last check to `a?.a?.a != null`.